delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/SupervisionSuite.scala

```diff
@@ -17,46 +18,91 @@ import scala.concurrent.duration._
 
+  private val expectedProgress = ProjectionProgress(Offset.at(20L), Instant.EPOCH, 20, 0, 0)
+
+  test("Watching restart projection restarts should be running") {
+    sv.describe(Supervisor.watchRestartMetadata.name)
+      .assertSome(
+        SupervisedDescription(
+          Supervisor.watchRestartMetadata,
+          ExecutionStrategy.EveryNode,
+          0,
+          ExecutionStatus.Running,
+          ProjectionProgress.NoProgress
+        )
+      )
+  }
+
   test("Ignore a projection when it is meant to run on another node") {
     for {
       flag      <- Ref.of[Task, Boolean](false)
       projection =
         CompiledProjection.fromStream(projection2, ExecutionStrategy.TransientSingleNode, evalStream(flag.set(true)))
       _         <- sv.run(projection).assert(ExecutionStatus.Ignored)
       _         <- Task.sleep(100.millis)
-      _         <- projectionStore.offset(projection2.name).assertNone
+      // The projection should still be ignored and should not have made any progress
+      _         <- sv.describe(projection2.name).assertSome(projection2Description)
+      // No progress has been saved in database either
+      _         <- projections.offset(projection2.name).assertNone
+      // This means the stream has never been started
       _         <- flag.get.assert(false)
     } yield ()
   }
 
-  test("Describe an ignored projection") {
-    sv.describe(projection2.name)
-      .assertSome(
-        SupervisedDescription(
-          projection2,
-          ExecutionStrategy.TransientSingleNode,
-          0,
-          ExecutionStatus.Ignored,
-          ProjectionProgress.NoProgress
-        )
-      )
+  test("Do nothing when attempting to restart a projection when it is meant to run on another node") {
+    for {
+      _ <- projections.scheduleRestart(projection2.name)
+      _ <- sv.describe(Supervisor.watchRestartMetadata.name)
+             .eventuallySome(
+               SupervisedDescription(
+                 Supervisor.watchRestartMetadata,
+                 ExecutionStrategy.EveryNode,
+                 0,
+                 ExecutionStatus.Running,
+                 // Watching restart projection has captured the restart and ignored
+                 ProjectionProgress(Offset.at(1L), Instant.EPOCH, 1, 1, 0)
+               )
+             )
+      _ <- sv.describe(projection2.name)
+             .eventuallySome(
+               SupervisedDescription(
+                 metadata = projection2,
+                 executionStrategy = ExecutionStrategy.TransientSingleNode,
+                 restarts = 0,
+                 status = ExecutionStatus.Ignored,
+                 progress = ProjectionProgress.NoProgress
+               )
+             )
+      // The restart has not been acknowledged and can be read by another node
+      _ <- projections.restarts(Offset.start).assertSize(1)
+    } yield ()
   }
 
   test("Cannot fetch ignored projection descriptions (by default)") {
```

Comment:
This test now fails because the `watch-restarts` projection is there and was previously unexpected. I think it's a similar issue with the other two failing tests in the suite.